src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const splittedStyles = sourceString.split(';');
+  const deletedInvalid = splittedStyles
+    .map(style => style.split(':'))
+    .filter(element => element.length !== 1);
+
+  deletedInvalid.forEach(style => {
+    stylesObject[style[0].trim()] = style[1].trim();
+  });
```

Comment:
forEach is the same as loop, let's use reduce method to create object from array